channels.go

```diff
@@ -52,7 +52,7 @@ func (c *Client) SearchChannels(params *SearchChannelsParams) (*SearchChannelsRe
 }
 
 type GetChannelInformationParams struct {
-	BroadcasterID string `query:"broadcaster_id"`
+	BroadcasterIDs []string `query:"broadcaster_id"` // Limit 100
```

Comment:
I'd prefer we not introduce this as a breaking change. Let's leave `BroadcasterID` as it is and introduce a new `BroadcasterIDs` field for handling this use case.

```suggestion
	// Deprecated: use BroadcasterIDs instead.
	BroadcasterID string `query:"broadcaster_id"`
	BroadcasterIDs []string `query:"broadcaster_id"` // Limit 100	
```